edxadapt/edxadapt.py

```diff
@@ -128,7 +128,11 @@ def get_course_id(self):
         # NOTE(idegtiarov) drop meaningless part of default course_id to decrease length of id which is stored in
         # the database
         return self.course_id.course.replace('course-v1:', '')
-
+    
+    # Decorate the view in order to support multiple devices e.g. mobile
+    # See: https://openedx.atlassian.net/wiki/display/MA/Course+Blocks+API
```

Comment:
@z4y4ts here it is: https://openedx.atlassian.net/wiki/display/EDUCATOR/Course+Blocks+API 
I will make another commit, thanks for noticing.